Description: some places failed to detect we build with no video
Author: Tzafrir Cohen <tzafrir@debian.org>

--- a/pjmedia/src/pjmedia-videodev/sdl_dev.c
+++ b/pjmedia/src/pjmedia-videodev/sdl_dev.c
@@ -21,6 +21,9 @@
 #include <pj/log.h>
 #include <pj/os.h>
 
+#if (! defined(PJMEDIA_HAS_VIDEO)) || (PJMEDIA_HAS_VIDEO == 0)
+#   undef PJMEDIA_VIDEO_DEV_HAS_SDL
+#endif
 #if defined(PJMEDIA_VIDEO_DEV_HAS_SDL) && PJMEDIA_VIDEO_DEV_HAS_SDL != 0
 #include <SDL.h>
 #include <SDL_syswm.h>
--- a/pjmedia/src/pjmedia-videodev/v4l2_dev.c
+++ b/pjmedia/src/pjmedia-videodev/v4l2_dev.c
@@ -25,6 +25,10 @@
 #include <pj/os.h>
 #include <pj/rand.h>
 
+#if (! defined(PJMEDIA_HAS_VIDEO)) || (PJMEDIA_HAS_VIDEO == 0)
+#   undef PJMEDIA_VIDEO_DEV_HAS_V4L2
+#endif
+#
 #if PJMEDIA_VIDEO_DEV_HAS_V4L2
 
 #include <linux/videodev2.h>
